Resources/Prototypes/Entities/Clothing/Head/sindicap.yaml

```
@@ -0,0 +1,13 @@
+- type: entity
+  parent: ClothingHeadBase
+  id: RepublicsSindiCap
+  name: кепка офицера синдиката
```

```suggestion
  name: кепка офицера синдиката
```
Эта штука должна быть через локализацию